app/templates/email/en_us/notifications/individual_merge_complete.jinja1

```diff
@@ -1,6 +1,10 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
looks like this one got **named** wrong (probably by me) with `jinja1` extension.  maybe we could fix this in this PR?  if not ... then maybe the next one from DEX-798.

Reply:
fixed in this pr.